x-pack/legacy/plugins/siem/public/components/links/index.tsx

```diff
@@ -239,11 +239,10 @@ const ReputationLinkComponent: React.FC<{
             {ipReputationLinks
               ?.slice(0, overflowIndexStart)
               .map(({ name, url_template: urlTemplate }: ReputationLinkSetting, id) => (
-                <EuiFlexItem grow={false}>
+                <EuiFlexItem grow={false} key={`reputationLink-${id}`}>
```

Comment:
I accidentally put key in its children, `ExternalLink` , moving to the proper place here.